server/apps/main/forms.py

```diff
@@ -63,6 +63,20 @@ class ShareExperienceForm(forms.Form):
     moderation_status = forms.ChoiceField(choices = statuses, widget = forms.Select(), required=False)
     moderation_status.group = "hidden"
 
+    authorship_choices = [
+        (True, "Experience is my own"),
+        (False, "Experience is someone else's"),
+    ]
+    first_hand_authorship = forms.ChoiceField(choices = authorship_choices, widget = forms.RadioSelect(), required=False)
+    first_hand_authorship.group = 4
+
+    authorship_relation = forms.CharField(label="Relationship",
+                                 max_length=300, strip=True, required=False,
+                                 widget=forms.TextInput(),
+                                 help_text='For example: "I am the non-autistic parent of an autistic adult", "I am the autistic parent of autistic children", "I am a teacher who works with autistic people", "I am best friends of an autistic person". - Please do not share personally identifying information.')
```

Comment:
This is not required on the form-side of things, but once someone picks "reporting on behalf of someone else" it should be mandatory i think. But that probably can't easily be verified inside the form setup here. :)

Reply:
Could do something like [this](https://docs.djangoproject.com/en/4.2/ref/forms/validation/#cleaning-and-validating-fields-that-depend-on-each-other) so it would get caught in validation?

Reply:
yes, that looks good!